tests/phpunit/tests/test-strings.php

```diff
@@ -289,4 +289,18 @@ public function test_import_empty_string() {
 		$mo->import_from_db( $lang );
 		$this->assertSame( array( 'test', 'test 2' ), array_keys( $mo->entries ) );
 	}
+
+	public function test_translate_untranslated_string_should_return_original_string() {
+		$pll_admin = new PLL_Admin( $this->links_model );
+		$pll_admin->init();
+
+		$lang = self::$model->get_language( 'en' );
+		$mo   = new PLL_MO();
+		$mo->add_entry( $mo->make_entry( 'test', '' ) );
+		$mo->export_to_db( $lang );
+
+		do_action( 'pll_language_defined' );
+
+		$this->assertSame( 'test', __( 'test', 'pll_string' ) ); // PHPCS:ignore WordPress.WP.I18n
```

Comment:
Wouldn't this be better? 
```suggestion
		$this->assertSame( 'test', pll__( 'test' ) );
```

Reply:
In fact, the issue #2368 was revealed in a admin context with a ACF compatibility issue where `__()` is callled instead of `pll__()`. It's why I chose this solution.
See 
https://github.com/polylang/polylang-pro/blob/new-acf-dev/integrations/acf/acf-translate-labels.php#L76-L78
and
https://github.com/AdvancedCustomFields/acf/blob/6.3.11/includes/api/api-helpers.php#L3103

I said to myself that this case could also happen in this way